fe/src/app/service/verify/components/Stepper.tsx

```diff
@@ -0,0 +1,100 @@
+"use client";
+
+import { ReactNode, useEffect, useState } from "react";
+import { Stepper as St, Step, Button } from "@/lib/materialTailwindExports";
+import {
+  FaCheck,
+  HiOutlineDocumentText,
+  IoDocumentAttachSharp,
+} from "@/lib/reactIconsExports";
+import InfoForm from "./InfoForm/InfoForm";
+import PdfForm from "./PdfForm";
+import { BASE_API_URL, SERVICE_STATUS_ROUTE } from "@/utils/urls";
+import { getToken } from "@/utils/getToken";
+
+const Stepper = ({ children }: { children: ReactNode }) => {
+  const [activeStep, setActiveStep] = useState(-1);
+  const [isLastStep, setIsLastStep] = useState(false);
+  const [isFirstStep, setIsFirstStep] = useState(false);
+  const [isForwardButtonDisabled, setIsForwardButtonDisabled] = useState(false);
+
+  const handleNext = () => !isLastStep && setActiveStep((cur) => cur + 1);
+  const handlePrev = () => !isFirstStep && setActiveStep((cur) => cur - 1);
+
+  useEffect(() => {
+    getServiceStatus();
+  }, []);
+
+  const getServiceStatus = async () => {
+    const token = await getToken();
+    const response = await fetch(`${BASE_API_URL}${SERVICE_STATUS_ROUTE}`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    const serviceStatus = await response.json();
+    console.log(serviceStatus);
+    if (serviceStatus === "Exists" || serviceStatus === "Denied") {
+      setIsForwardButtonDisabled(true);
+      setActiveStep(0);
+    } else if (serviceStatus === "CreatedInDataBase") {
+      setActiveStep(1);
+    } else if (serviceStatus === "Pending") {
+      setActiveStep(2);
+    }
+  };
+  return (
+    <>
+      <St
+        className="z-0"
+        activeStep={activeStep}
+        isLastStep={(value) => setIsLastStep(value)}
+        isFirstStep={(value) => setIsFirstStep(value)}
+      >
+        <Step>
+          <HiOutlineDocumentText />
+        </Step>
+        <Step>
+          <IoDocumentAttachSharp />
+        </Step>
+        <Step>
+          <FaCheck />
+        </Step>
+      </St>
+      <div className="shadow-lg flex flex-col justify-center rounded-xl my-5 p-4 min-h-">
+        {activeStep == -1 && (
+          <h2 className="text-3xl font-bold py-10 px-4 text-center">
+            Loading...
+          </h2>
+        )}
+        {activeStep == 0 && (
```

Comment:
Could add enum, so it would make more sense when equaling to enum type instead of number. Why not using triple equal?